<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1021</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{background-color:transparent;}
QTabWidget::pane{border:2px;}
QTabWidget::tab-bar{alignment:left;}
QTabBar::tab{background:rgb(25, 150, 0);
                color:white;
                min-width:20ex;
                min-height:8ex;
				
				border-bottom-style:1px;
                }
QTabBar::tab:hover{background:grey;
                color:white;
                }
QTabBar::tab:selected{
                border-color: black;
                background:rgb(225, 225, 225, 50);
				border:1px groove rgb(25, 109, 0);
				border-bottom-style:0px;
                color:black;
                }</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="PluginsManager">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1011</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="logPlainTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>480</y>
        <width>1011</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>650</y>
        <width>1011</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="FullSelectBtn">
       <property name="geometry">
        <rect>
         <x>264</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
      <widget class="QPushButton" name="DownloadBtn">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>下载/更新</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
